templates/zerver/app/navbar.html

```diff
@@ -171,11 +171,19 @@
                                 </a>
                             </li>
                             {% endif %}
+                            {% if page_params.is_web_public_visitor %}
+                            <li role="presentation">
+                                <a href="/login" class="logout_button" role="menuitem">
```

Comment:
`logout_button` has a click handler; do we want to remove that class here?

Reply:
Nice catch, fixed.